core-v1/src/main/scala/data/Justified.scala

```diff
@@ -0,0 +1,35 @@
+package com.rallyhealth.vapors.v1
+
+package data
+
+sealed abstract class Justified[+V] {
```

Comment:
Since this isn't used anywhere that my IDE can detect, I think that we need some more docs on this.

Reply:
This is used in a future PR. I think I add docs on that PR as well. Sorry for the messy scattershot on implementing this.